pkg/util/issuer.go

```diff
@@ -14,16 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package acme
+package util
 
 import (
-	"context"
-	"errors"
-
 	"github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha1"
-	"github.com/jetstack/cert-manager/pkg/issuer"
 )
 
-func (a *Acme) Sign(ctx context.Context, cr *v1alpha1.CertificateRequest) (*issuer.IssueResponse, error) {
-	return nil, errors.New("sign not implemented by ACME issuer")
+// issuerKind returns the kind of issuer for a certificate
+func IssuerKind(ref v1alpha1.ObjectReference) string {
+	if ref.Kind == "" {
+		return v1alpha1.IssuerKind
+	}
+	return ref.Kind
```

Comment:
Can you put this in `pkg/api/util` and not `pkg/util`? This is specifically API related so IMO makes sense to namespace it to `api`?